recipes-dotnet/corehost/coreclr_2.0-beta.bb

```diff
@@ -54,7 +53,18 @@ do_install() {
 	install -m 0755 ${src}/mscorlib.dll ${target}
 	install -m 0755 ${src}/System.Private.CoreLib.dll ${target}
 	install -m 0755 ${src}/SOS.NETCore.dll ${target}
+
+	# Create dev package
+	install -d ${D}/opt/dotnet-dev/
+	for i in `ls ${src}/.nuget/pkg/*.nupkg`
+	do
+		install -m 644 ${i} ${D}/opt/dotnet-dev/
```

Comment:
Ah, that I have. I was expecting something along the lines of https://www.nuget.org/packages/System.Linq/
but this seems to have changed. So now I've come full circle to what @JensNordenbro said in https://github.com/Tragetaschen/meta-aspnet/pull/5#issuecomment-280667311